diffpy/snmf/files.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
```

Comment:
let's change the module name (and the test name too....) to `io.py` and put all the io handlers and parsers in there.